caffe2/onnx/backend.cc

```diff
@@ -304,7 +319,7 @@ Caffe2Backend::get_renamed_operators() const {
 const std::unordered_map<std::string, std::string>&
 Caffe2Backend::get_renamed_attrs() const {
   const static std::unordered_map<std::string, std::string> kRenamedAttrs{
-      {"kernel_shape", "kernels"}};
+      {"kernel_shape", "kernels"}, {"auto_pad", ""}};
```

Comment:
I think you don't need this, in the `CreateConvePoolOpBase` you can remove the `auto_pad` attribute before calling `CommonOnnxNodeToCaffe2Ops`.

Reply:
Sure, done :-)